include/ola/Clock.h

```diff
@@ -222,7 +222,11 @@ class Clock {
  public:
   Clock() {}
   virtual ~Clock() {}
+  // Get the current monotonic clock if available,
+  // otherwise falls back to wall clock time.
   virtual void CurrentTime(TimeStamp *timestamp) const;
+  // Get wall clock time.
+  virtual void CurrentRealTime(TimeStamp *timestamp) const;
```

Comment:
I wonder if CurrentWallClockTime might be a bit less ambiguous, did there seem to be any consistency online when you were looking for stuff?

Reply:
I went with CurrentRealTime based on `clock_gettime`'s terminology, although they make it a point to clarify it means wall-clock time. The man page states: 
```
CLOCK_REALTIME
              System-wide  clock that measures real (i.e., wall-clock) time. ...
```
The `time` function also uses the term "real".  My opinion is stick with CurrentRealTime, but provide clarification in the documentation.

Reply:
Okay that makes sense to me, I'm happy to stick with Real.